docs/topic/idle-culler.rst

```diff
@@ -0,0 +1,86 @@
+.. _topic/idle-culler:
+
+=============================
+Culling idle notebook servers
+=============================
+
+The idle culler is a hub-managed service that automatically shuts down idle
```

Comment:
We should split this into a couple parts, written as paragraphs / sections.


1. What idle culling does & why. Something like:

```
The idle culler automatically shuts down user notebook servers when they have
not been used for a certain time period to reduce total resource usage on
your JupyterHub. 
```

2. What constitutes 'idle'? When is that triggered? This should be a small section of its own.

3. What are the default settings? Should also be a section. It should be descriptive rather than just listing the settings themselves.

Reply:
We can iterate on these as we go forward and I'm totally happy to help write them too if you wish :)

Reply:
Thanks a lot for your feedback @yuvipanda :heart:
I tried my best to address you suggestions, but I have a feeling the docs might also need your intervention :eyes:

Reply:
@GeorgianaElena This is perfect! I'm going to merge this now.